docs/source/serving/update.rst

```diff
@@ -49,54 +63,65 @@ We can use :code:`sky serve status http-server` to check the status of the servi
+    NAME         VERSION  UPTIME  STATUS  REPLICAS  ENDPOINT
+    http-server  1        1m 41s  READY   2/2       44.206.240.249:30002
 
     Service Replicas
-    SERVICE_NAME  ID  VERSION  IP              LAUNCHED    RESOURCES       STATUS  REGION     
-    http-server   1   1        54.173.203.169  2 mins ago  1x AWS(vCPU=2)  READY   us-east-1  
-    http-server   2   1        52.87.241.103   2 mins ago  1x AWS(vCPU=2)  READY   us-east-1 
+    SERVICE_NAME  ID  VERSION  IP              LAUNCHED    RESOURCES       STATUS  REGION
+    http-server   1   1        54.173.203.169  2 mins ago  1x AWS(vCPU=2)  READY   us-east-1
+    http-server   2   1        52.87.241.103   2 mins ago  1x AWS(vCPU=2)  READY   us-east-1
 
-Service ``http-server`` has an initial version of 1. Suppose we want to update the service to use 4 CPUs instead of 2, we can update the task yaml ``examples/serve/http_server/task.yaml``, by changing the ``cpu`` parameter from 2 to 4. We can then use :code:`sky serve update` to update the service:
+Service ``http-server`` has an initial version of 1.
 
-That is, we update the cpus field of the YAML file from 2 to 4:
+Suppose we want to update the service to use 4 vCPUs instead of 2. We can update
+the task yaml ``examples/serve/http_server/task.yaml``, by changing the ``cpu``
+field:
 
 .. code-block:: yaml
-    :emphasize-lines: 8
+    :emphasize-lines: 10
 
     # examples/serve/http_server/task.yaml
     service:
-        readiness_probe: /
-        replicas: 1
+      readiness_probe:
+        path: /health
+        initial_delay_seconds: 20
+      replicas: 2
 
     resources:
-        ports: 8081
-        cpus: 4+
+      ports: 8081
+      cpus: 4+
 
-    workdir: .
+    workdir: examples/serve/http_server
 
     run: python3 server.py
 
+We can then use :code:`sky serve update` to update the service:
 
 .. code-block:: console
 
     $ sky serve update http-server examples/serve/http_server/task.yaml
 
-SkyServe will first launch two new replicas with 4 CPUs. When the number of new replicas reaches the ``min_replicas`` (i.e., 2) required for the service, SkyServe will scale down old replicas to save cost. The service's version is updated from 1 to 2. The replicas with ID 3 and 4 are the new replicas with 4 CPUs. The replicas with ID 1 and 2 are the old replicas with 2 CPUs. When the new replicas are still provisioning, SkyServe will only send traffic to the old replicas.
+SkyServe will first launch two new replicas with 4 vCPUs. When the number of new
+replicas reaches the ``min_replicas`` (set to ``service.replicas`` when unspecified; i.e., 2) required for the service,
+SkyServe will scale down old replicas to save cost. The service's version is
+updated from 1 to 2. Replicas 3 and 4 are the new replicas with 4
+vCPUs. Replicas 1 and 2 are the old replicas with 2 vCPUs. When the
+new replicas are still provisioning, SkyServe will only send traffic to the old
+replicas.
```

Comment:
Change to sth like "When the new replicas are not reached min_replicas"? There is some case when one replica is still provisioning while other is ready

Reply:
Reworded and reordered some sentences.